VERSION

```diff
@@ -1,4 +1,4 @@
 COLUMNSTORE_VERSION_MAJOR=1
 COLUMNSTORE_VERSION_MINOR=4
 COLUMNSTORE_VERSION_PATCH=3
-COLUMNSTORE_VERSION_RELEASE=1
+COLUMNSTORE_VERSION_RELEASE=2
```

Comment:
we'll need to bump the version in dev-1.4 anyway.  ignore this intermediate update